docs/indepth/installation.rst

```diff
@@ -347,10 +347,37 @@ You must also install the ZFS package repository.
 The following commands will install the two repositories and the ``clusterhq-flocker-node`` package.
 Paste them into a root console on the target node:
 
-.. task:: install_flocker
+.. task:: install_flocker fedora-20
    :prompt: [root@node]#
 
-Installing ``clusterhq-flocker-node`` will automatically install Docker, but the ``docker`` service may not have been enabled or started.
+Installing on CentOS 7
+----------------------
+
+Flocker requires the latest available kernel.
```

Comment:
Oh.  But there is no distro-testing for CentOS 7.  Did we look around for an alternative to that and not find anything?  That would be sad but maybe not much we can do about it...